src/main/resources/public/js/app.js

```diff
@@ -64,6 +64,21 @@ function fire(row, column) {
         displayGameState(currModel);
         gameModel = currModel;
 
+        setTimeout( function(){
```

Comment:
I didn't believe you at first, but you're right. This doesn't happen to me in Firefox, but it does in Chrome. I'm gonna try to figure out why this is the case, but for now I'm content with leaving it in.